src/typings.d.ts

```diff
@@ -151,6 +151,13 @@ interface ClipParams {
    *  - See:      https://tonejs.github.io/docs/13.8.25/Volume
    */
   volume?: number;
+  /**
+   * Boolean parameter to trigger offline rendering. Use this to help
```

Comment:
sorry for the nit but is the sentence incomplete accidentally?

Reply:
Sure. 😅 Thanks for seeing that.

Reply:
Solved in https://github.com/scribbletune/scribbletune/pull/113/commits/1ff91750cb3dad5b79bff216af47fcc83b9f781c